examples/discovery-search-app/cypress/integration/autocomplete/spec.ts

```diff
@@ -31,7 +31,7 @@ describe('Autocomplete', () => {
       cy.wait('@getAutocompletions');
     });
 
-    it('displays a dropdown of autocomplete suggestions', () => {
+    it.only('displays a dropdown of autocomplete suggestions', () => {
```

Comment:
guessing this is temporary?

Reply:
Whoops, yeah, that needs to get axed. I think I added that during the demo I gave, then forgot to remove it, because I wasn't touching the autocomplete tests. Removing...